src/Raven.Studio/typescript/components/pages/database/settings/revisionsBinCleaner/RevisionsBinCleaner.tsx

```diff
@@ -0,0 +1,182 @@
+                                                        type="number"
+                                                        disabled={
+                                                            !hasDatabaseAdminAccess ||
+                                                            formState.isSubmitting ||
+                                                            !formValues.isMinimumEntriesAgeToKeepEnabled
+                                                        }
+                                                        addon="minutes"
+                                                    />
+                                                </Collapse>
+                                            </div>
+                                            <div>
+                                                <FormSwitch
+                                                    name="isRefreshFrequencyEnabled"
+                                                    control={control}
+                                                    color="primary"
+                                                    className="mb-3"
+                                                    disabled={
+                                                        !hasDatabaseAdminAccess ||
+                                                        formState.isSubmitting ||
+                                                        !formValues.isRevisionsBinCleanerEnabled
+                                                    }
+                                                >
+                                                    Set custom refresh frequency
+                                                </FormSwitch>
+                                                <FormInput
+                                                    name="refreshFrequencyInSec"
+                                                    control={control}
+                                                    type="number"
+                                                    disabled={
+                                                        !hasDatabaseAdminAccess ||
+                                                        formState.isSubmitting ||
+                                                        !formValues.isRefreshFrequencyEnabled
+                                                    }
+                                                    placeholder="Default (300)"
+                                                    addon="seconds"
+                                                />
+                                            </div>
+                                        </div>
+                                    </CardBody>
+                                </Card>
+                            </Col>
+                        </Form>
+                    </Col>
+                    <Col sm={12} lg={4}>
+                        {false && <RevisionsBinCleanerInfoHub />}
```

Comment:
?

Reply:
to clarify, Until Danielle adds the text, this component will remain disabled so as not to block the possibility of it being merged.

I have added a comment above with this information.
```suggestion
                        {/*TODO: Until Danielle adds the text, this component will remain disabled so as not to block the possibility of it being merged.*/}
                        {false && <RevisionsBinCleanerInfoHub />}
```